cloud/forking.rst

```diff
@@ -0,0 +1,26 @@
+Forking
+#######
+
+Forking a Citus Cloud formation makes a copy of the cluster's data at an instant in time and produces a new formation in precisely that state. It allows you to change, query, or generally experiment with production data in a separate protected environment. Fork creation runs quickly, and you can do it as often as you want without causing any extra load on the original cluster. This is because forking doesn't query the cluster, rather it taps into the write-ahead logs for each database in the formation.
+
+How to Fork a Formation
+-----------------------
+
+Citus Cloud makes forking easy. The control panel for each formation has a "Fork" tab. Go there and enter the name, region, and node sizing information for the destination cluster.
+
+.. image:: ../images/cloud-fork.png
+
+Shortly after you click "Fork Formation," the new formation will appear in the Cloud console. It runs on separate hardware and your database can connect to it in the :ref:`usual way <connection>`.
+
+When is it Useful
```

Comment:
I'm also OK to have a more abbreviated version of the "under the hood" section previously.